catalog/spatial/ogc/spatial-ogc-common/src/test/java/org/codice/ddf/spatial/ogc/catalog/common/TestTrustedRemoteSource.java

```diff
@@ -163,6 +145,16 @@ public static void tearDownAfterClass() {
     /**
      * Tests that server properly accepts trusted certificates.
      */
+//    @Test
```

Comment:
@ricklarsen did you mean to just remove this commented out test? Looks like you have it below as well.